packages/views/profile_detail.jade

```diff
@@ -0,0 +1,10 @@
+template(name="profileDetail")
+  .container
+    .row
+      .col-sm-12
+        .profile-detail
+          p Full Name: #{userProfile.fullName}
+          p Job Title: #{userProfile.jobTitle}
+          p Bio: #{userProfile.bio}
+          unless userProfile.emailHidden
```

Comment:
@aslagle I'm assuming this isn't an effective way to hide the email address and that we need to exclude the email from the subscription?